tests/test_hybrid.py

```diff
@@ -278,3 +278,74 @@ def test_sap_fem_vs_robot(show_viewer):
 
     # Check that the legs of the ants are resting on the sphere
     assert_allclose(robot.get_qpos()[-4:].abs(), 1.0, tol=0.1)
+
+
+@pytest.mark.required
+@pytest.mark.parametrize("substeps", [1, 10])
+def test_rigid_mpm_legacy_coupling(substeps, show_viewer):
+    # This test is aimining at two things:
+    # 1) When substep = 1, the rigid object should be affected by the MPM particles
+    # 2) Regardless of substeps, as far as the total step dt is the same, they should give consistent results
+    dt_per_substep = 4e-4
+    horizon_substeps = 2000
```

Comment:
let's see how long will this test run.

Reply:
No. This is not a good approach. This is the reason why running the unit tests now takes 5h30 on windows without cache. You MUST design your test to be as short as possible. Otherwise it will seriously hinder our capability to add more unit tests in the future.